smbios.adoc

```diff
@@ -16,24 +16,23 @@ NOTE: The structures and fields in this section are defined in a manner consiste
 [%header, cols="5,25"]
 |===
 | ID#     ^| Requirement
-| `SMBIOS_010` | A Baseboard/Module Information (Type 02) structure SHOULD be implemented.
```

Comment:
Why did you drop Type 02 here?

Reply:
The reason the SMBIOS Table 02 requirement was omitted in this patch is that we have specified in SMBIOS_010 to adhere to DMTF SMBIOS requirements. Within SMBIOS_010, it is explicitly stated that these tables must be implemented if the hardware is supported. Therefore, I consider it unnecessary to keep it in the table.